plugins/woocommerce/tests/e2e-pw/tests/basic.spec.js

```diff
@@ -2,7 +2,8 @@ const { test, expect } = require( '@playwright/test' );
 const { logIn } = require( '../utils/login' );
 const { admin, customer } = require( '../test-data/data' );
 
-test( 'Load the home page', async ( { page } ) => {
+test( 'Load the home page', async ( { page }, testInfo ) => {
+	expect( testInfo.retry ).toBe( 1 );
```

Comment:
Test change to force a flaky test. To revert before merging.

```suggestion
test( 'Load the home page', async ( { page } ) => {
```